backend/python/tapa/tapac.py

```diff
@@ -303,6 +304,9 @@ def main(argv: Optional[List[str]] = None):
   logging.getLogger().setLevel(logging_level)
   _logger.info('logging level set to %s', logging.getLevelName(logging_level))
 
+  version = pkg_resources.get_distribution("tapa").version
```

Comment:
The package metadata does not always match the running code, especially when installed with `--editable`. Let's use method 4 in https://packaging.python.org/en/latest/guides/single-sourcing-package-version/. I'll rebase and force-push to this branch shortly.